mbed-hal-efm32/serial_api_HAL.h

```diff
@@ -0,0 +1,45 @@
+/***************************************************************************//**
```

Comment:
Is indeed useless, used to contain headers for the dynamic peripheral allocation.